diff --git a/src/app/dashboard/document/data-documentEmployees.tsx b/src/app/dashboard/document/data-documentEmployees.tsx
index 145f8aa..ea53b22 100644
--- a/src/app/dashboard/document/data-documentEmployees.tsx
+++ b/src/app/dashboard/document/data-documentEmployees.tsx
@@ -46,6 +46,7 @@ import { Url } from 'next/dist/shared/lib/router/router'
 import { useLoggedUserStore } from '@/store/loggedUser'
 import { DataTable } from '../employee/data-table'
 import { useSidebarOpen } from '@/store/sidebar'
+import { MdOutlineRemoveRedEye } from 'react-icons/md'
 
 interface DataDocumentsEmployeesProps<TData, TValue> {
   columEmp: ColumnDef<TData, TValue>[] | any
@@ -407,11 +408,14 @@ export function DataDocumentsEmployees<TData, TValue>({
                       >
                         {cell.column.id === 'document_url' ? (
                           <Link href={cell.getValue() as any} target="_blank">
-                            <img
+                            <MdOutlineRemoveRedEye
+                              style={{ width: '24px', height: '24px' }}
+                            />
+                            {/* <img
                               src={cell.getValue() as any}
                               alt="Foto"
                               style={{ width: '68px', height: '68px' }}
-                            />
+                            /> */}
                           </Link>
                         ) : (
                           flexRender(
diff --git a/src/app/dashboard/document/data-documentsEquipment.tsx b/src/app/dashboard/document/data-documentsEquipment.tsx
index b403406..e71b96f 100644
--- a/src/app/dashboard/document/data-documentsEquipment.tsx
+++ b/src/app/dashboard/document/data-documentsEquipment.tsx
@@ -46,7 +46,7 @@ import { Url } from 'next/dist/shared/lib/router/router'
 import { useLoggedUserStore } from '@/store/loggedUser'
 import { DataTable } from '../employee/data-table'
 import { useSidebarOpen } from '@/store/sidebar'
-
+import { MdOutlineRemoveRedEye } from 'react-icons/md'
 interface DataDocumentsEquipmentProps<TData, TValue> {
   columns: ColumnDef<TData, TValue>[] | any
   data: TData[]
@@ -207,7 +207,7 @@ export function DataDocumentsEquipment<TData, TValue>({
             <SelectContent>
               <SelectGroup>
                 <SelectLabel>Filas por página</SelectLabel>
-                {maxRows.map((option: string,index:number) => (
+                {maxRows.map((option: string, index: number) => (
                   <SelectItem key={index} value={option}>
                     {option}
                   </SelectItem>
@@ -391,11 +391,14 @@ export function DataDocumentsEquipment<TData, TValue>({
                       >
                         {cell.column.id === 'document_url' ? (
                           <Link href={cell.getValue() as any} target="_blank">
-                            <img
+                            <MdOutlineRemoveRedEye
+                              style={{ width: '24px', height: '24px' }}
+                            />
+                            {/* <img
                               src={cell.getValue() as any}
                               alt="Foto"
                               style={{ width: '68px', height: '68px' }}
-                            />
+                            /> */}
                           </Link>
                         ) : (
                           flexRender(
diff --git a/src/components/SideLinks.tsx b/src/components/SideLinks.tsx
index d0d0c57..62c10f6 100644
--- a/src/components/SideLinks.tsx
+++ b/src/components/SideLinks.tsx
@@ -12,7 +12,7 @@ import {
   MdOutlinePhoneIphone,
   MdOutlineSpaceDashboard,
 } from 'react-icons/md'
-
+import { FaTruckMonster } from 'react-icons/fa'
 export async function getServerSideProps(context: any) {
   const { params } = context
   const { type } = params
@@ -39,7 +39,7 @@ const links = [
   {
     name: 'Equipos',
     href: '#',
-    icon: <MdOutlinePhoneIphone size={sizeIcons} />,
+    icon: <FaTruckMonster size={sizeIcons} />,
     submenu: [
       { name: 'Todos', href: '/dashboard/equipment?type=Todos' },
       { name: 'Vehículos', href: '/dashboard/equipment?type=1' },
